src/services/onramper/donationService.ts

```diff
@@ -140,6 +141,7 @@ export const createFiatDonationFromOnramper = async (
       fiatTransaction.payload.outCurrency,
       priceChainId,
       fiatTransaction.payload.outAmount,
+      ChainType.EVM,
```

Comment:
Let's make ChainType.EVM the default value and revert changes in other similar places.